kev/backends/dynamodb/db.py

```diff
@@ -0,0 +1,89 @@
+import decimal
+
+import boto3
+from boto3.dynamodb.conditions import Key
+from botocore.exceptions import ClientError
+
+from kev.backends import DocDB
+from kev.exceptions import DocNotFoundError
+
+
+class DynamoDB(DocDB):
+
+    db_class = boto3.resource
```

Comment:
There is the same class attribute in other backends, I thought that it's necessary.